lib/ReactTags.js

```diff
@@ -203,6 +204,25 @@ ReactTags.propTypes = {
   suggestions: PropTypes.array,
   autofocus: PropTypes.bool,
   autoresize: PropTypes.bool,
+  delimiters: function (props, propName, componentName) {
```

Comment:
yeah, I was hoping to keep the array proptype in somehow, but if the idea was to test for an empty array the entire thing needs validating =(